src/mesomatic/types.clj

```diff
@@ -99,12 +99,12 @@
 
 (defmethod pb->data Protos$Status
   [^Protos$Status status]
-  (cond
-    (= status Protos$Status/DRIVER_RUNNING)     :driver-running
-    (= status Protos$Status/DRIVER_NOT_STARTED) :driver-not-started
-    (= status Protos$Status/DRIVER_ABORTED)     :driver-aborted
-    (= status Protos$Status/DRIVER_STOPPED)     :driver-stopped
-    :else status))
+  (case status
```

Comment:
I think `case` doesn't work like this for enum values, as they are not compile time literals. 

see:

```
(case java.util.concurrent.TimeUnit/DAYS java.util.concurrent.TimeUnit/DAYS 1 **:broken)

```

You could compare ordinal enum values, that's something often macro'ified (https://github.com/mpenet/commons/blob/master/src/clj/qbits/commons/enum.clj#L28-L38). 

In the same file I mentioned I have utils to automate this kind of things (ex create a fn that takes clojure style enum keyword (lowercased "_/-" replaced, etc) and throw with a meaningful error message), feel free to lift these functions if you need them. 

you can see it in use here: https://github.com/mpenet/alia/blob/master/modules/alia/src/qbits/alia/enum.clj